packages/components/src/utils/validation.ts

```diff
@@ -38,6 +38,20 @@ const isValidOneMapEmbedUrl = (urlObject: URL) => {
   )
 }
 
+// Slug pattern obtained from:
+// https://github.com/opengovsg/maps/blob/8c3f2e6c9af1be78fac5fc69115858241ae1f720/apps/web/modules/map/schema.ts#L3-L16
+const OGP_MAPS_SLUG_REGEX_PATTERN = "[a-z0-9][a-z0-9_-]*[a-z0-9]"
```

Comment:
should we be doing the check here on what is a valid or invalid slug?

seems like the intention is to have valid URL for the embedding, but
1. we are duplicating the application logic from Maps, and
2. this is half the validation (doesn't validate the other regex and min/max strings) so feels neither here nor there

Reply:
Updated in 7beee0e55a2c00faf56a996a8928089c40df1c7c.